[CN-1306] add docs for workload identity (#222)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/backup-restore.adoc

include::ROOT:example$/hot-backup-external-aws.yaml[]
----

> NOTE: Please see xref:authorization.adoc[Authorization Methods to Access Cloud Provider Resources] page to learn more about accessing resources on different cloud providers.

Reviewer: ```suggestion
NOTE: For further information about accessing resources on different cloud providers, see xref:authorization.adoc[Authorization Methods to Access Cloud Provider Resources].
```

include::ROOT:example$/hot-backup-external-gcp.yaml[]
----

> NOTE: Please see xref:authorization.adoc[Authorization Methods to Access Cloud Provider Resources] page to learn more about accessing resources on different cloud providers.

Reviewer: ```suggestion
NOTE: For further information about accessing resources on different cloud providers, see xref:authorization.adoc[Authorization Methods to Access Cloud Provider Resources].
```

include::ROOT:example$/hot-backup-external-azure.yaml[]
----

> NOTE: Please see xref:authorization.adoc[Authorization Methods to Access Cloud Provider Resources] page to learn more about accessing resources on different cloud providers.

Reviewer: ```suggestion
NOTE: For further information about accessing resources on different cloud providers, see xref:authorization.adoc[Authorization Methods to Access Cloud Provider Resources].
```
